apps/api/src/migration/1678711953274-ChangeApplicantIdColumnToVarChar.ts

```diff
@@ -0,0 +1,13 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class ChangeApplicantIdColumnToVarChar1678711953274 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`ALTER TABLE "ien_applicants" DROP COLUMN "applicant_id"`);
```

Comment:
Applicant_id and ats1_id come together from ATS. Considering we take applicant_id as a unique `id,` I wonder if it's necessary to check the uniqueness of `ats1_id`. It seems out of scope to validate the consistency of ATS data.

Reply:
I don't see any harm in keeping it unique as well, under the assumption they're also using it as a unique identifier between ATS and IEN.  @dbayly-freshworks is this correct or at least the intent?